AO3-7077 update tag wranglings index page

anmazz opened the pull request:
# Pull Request Checklist

<!-- Mark steps in the checklist as complete by changing `[ ]` to `[X]` -->
* [ ] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [ ] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [ ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [ ] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [ ] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-7077

## Purpose

Updates and internationalizes tag wranglers info page



## Credit

anna (she/her)

sarken commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,100 @@
-      <li><a href="http://wiki.transformativeworks.org/mediawiki/Category:Fandoms_%26_Megafandoms">Fandom &amp; Megafandom notes</a></li>
+      <li>
+        <%= t(".general_resources.tag_wrangling_committee_html",
+          link: link_to(t(".general_resources.tag_wrangling_committee_link"), 
```
Yes, please make sure to follow our [i18n guidelines for hyperlinks](https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards#hyperlinks). Our translators won't have any idea what link text is if the variable doesn't correspond with the key for the linked text.

Bilka2 commented on config/locales/views/en.yml:
```diff
@@ -2075,6 +2075,46 @@ en:
+      notes:
+        chair_contact_html: For confidential issues (including difficulty reaching other wranglers), please email Chairs at %{tag_wranglers_chair_email} or use the phrase <strong>"Question for a Chair!"</strong> in %{tw_wranglers_slack_link}, %{tw_wranglers_quietroom_slack_link}, or in any training room and let them know you would like to chat via direct message.
+        landmark_heading: Notes
+        supervisor_contact_html: If you have questions, difficulty accessing the wiki, or need technical assistance, please email Tag Wranglers Supervisors or use the phrase <strong>"Question for a Chair!"</strong> to contact a Supervisor in %{tw_wranglers_slack_link}, %{tw_wranglers_quietroom_slack_link}, or in any training room.
```
The guidelines are a bit unclear about this right now, but it would be nice to split out the `strong` text in here into a separate variable as well so the strong tag can be added in Ruby instead of here. You can find a lot of examples of this in the TOS, e.g. `t(".applicability.transfers_necessary_html", consent_to_us_processing_bold: tag.strong(t(".applicability.consent_to_us_processing")))`

Furthermore, text in the quotes is incorrect (and the quotes were also moved around). Please exactly match the text given in the Jira issue

Bilka2 commented on config/locales/views/en.yml:
```diff
@@ -2075,6 +2075,46 @@ en:
+      general_resources_heading: 'General Wrangling Resources:'
+      heading: Tag Wrangling
+      notes:
+        chair_contact_html: For confidential issues (including difficulty reaching other wranglers), please email Chairs at %{tag_wranglers_chair_email} or use the phrase <strong>"Question for a Chair!"</strong> in %{tw_wranglers_slack_link}, %{tw_wranglers_quietroom_slack_link}, or in any training room and let them know you would like to chat via direct message.
```
```suggestion
        chair_contact_html: For confidential issues (including difficulty reaching other wranglers) please email Chairs at %{tag_wranglers_chair_email} or use the phrase "<strong>Question for a Chair!</strong>" in %{tw_wranglers_slack_link}, %{tw_wranglers_quietroom_slack_link}, or in any training room and let them know you would like to chat via direct message.
```
Please match the text given in the Jira issue exactly. The note regarding extracting the strong tag also applies here

If the text in the Jira issue and the HTML in the Jira issue disagree, please pick the HTML version

Bilka2 commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,84 @@
+      </li>
     </ul>
-    <h3 class="heading">Additional Resources:</h3>
+    <h3 class="heading"><%= t(".general_resources_heading") %></h3>
```
Please nest this into `general_resources` as well

Bilka2 reviewed (changes requested):
Please do not request reviews from specific people, we review pull requests as time and priorities permit

anmazz commented:
> Please do not request reviews from specific people, we review pull requests as time and priorities permit

My bad, wasn’t a specific request for sarken, just a signal this as “ready to be reviewed again” since this PR wasn’t labeled that comments had been added/I addressed some comments. Used to doing that at work, will try to remember not to do that again lol

Hamham6 commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,84 @@
+              tag_wrangling_committee_link: link_to(t(".general_resources.tag_wrangling_committee_link_label"), "http://wiki.transformativeworks.org/mediawiki/Category:Tag_Wrangling_Committee")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.tag_wrangler_training_html",
```
Noting here that the Jira wording has "Tag Wrangling Training", altho I'm not sure how much the difference matters when it's being used to describe something else.

Hamham6 reviewed (commented):
Just some tweaking for key names!

anmazz commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,84 @@
-      <li>Make tags canonical according to the <%= link_to 'wrangling guidelines', wrangling_guidelines_path %>.</li>
-      <li>If you have trouble accessing the wiki, please <a href="mailto:tagwranglers-personnel@transformativeworks.org">email tag wranglers personnel</a>. For any other questions relating to tag wrangling, please <a href="mailto:tagwranglers-staff@transformativeworks.org">email wrangling staff</a>, or for confidential issues please <a href="mailto:tagwranglers-chair@transformativeworks.org">emai ...
+      <li>
+        <%= t(".notes.wrangle_guidelines_html", wrangling_guidelines_link: link_to(t(".notes.wrangling_guidelines_link_label"), wrangling_guidelines_path)) %>
```
@Bilka2  I found an example in another en.yml I was following as an example (written by sarken)
https://github.com/otwcode/otwarchive/blob/master/config/locales/views/en.yml#L864-L865

I can change it, I found link_label/html/link to be more clear. I think if the link label is different, especially for a page like this, it's going to be a bit confusing. Would love to hear your thoughts.

anmazz commented:
@Bilka2 thanks for the review, I think I got everything. This one's been really tedious so sorry about that, been hard for me to catch the little mistakes. Incorporated some of @Hamham6's notes as well but not all.

Bilka2 commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,84 @@
 <div class="tag wrangling home">
   <!--Descriptive page name, messages and instructions-->
-  <h2 class="heading"><%= ts('Tag Wrangling') %></h2>
+  <h2 class="heading"><%= t(".heading") %></h2>
```
agreed, please make this `page_heading` to standardize

Bilka2 commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,84 @@
+  <h2 class="heading"><%= t(".heading") %></h2>
   <% unless params[:show] %>
-    <h4 class="landmark heading"><%= ts('Notes') %></h4>
+    <h4 class="landmark heading"><%= t(".notes.landmark_heading") %></h4>
```
I'd go with just `.landmark` here

Bilka2 commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,88 @@
-      <li>Make tags canonical according to the <%= link_to 'wrangling guidelines', wrangling_guidelines_path %>.</li>
-      <li>If you have trouble accessing the wiki, please <a href="mailto:tagwranglers-personnel@transformativeworks.org">email tag wranglers personnel</a>. For any other questions relating to tag wrangling, please <a href="mailto:tagwranglers-staff@transformativeworks.org">email wrangling staff</a>, or for confidential issues please <a href="mailto:tagwranglers-chair@transformativeworks.org">emai ...
+      <li>
+        <%= t(".notes.wrangling_guidelines_html",
+              wrangling_guidelines_link: link_to(t(".notes.wrangling_guidelines"), wrangling_guidelines_path)) %>
```
For the note section to be consistent within all three points, the variable for the overall string here should also be different from the link variable, so something like `wrangle_to_guidelines_html` (while keeping the link the same) would be better

Bilka2 commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,84 @@
+              tag_wrangling_slack_tutorial_link: link_to(t(".general_resources.tag_wrangling_slack_tutorial_link_label"), "https://wiki.transformativeworks.org/mediawiki/Tag_Wrangling_Slack_Tutorial")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.fandoms_in_need_html",
```
agreed

Bilka2 commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,88 @@
+      <li>
+        <%= t(".general_resources.ao3_catch_and_release_html",
+              ao3_catch_and_release_link: link_to(t(".general_resources.ao3_catch_and_release"), tag_wrangler_path("Ao3_Catch_and_Release")),
+              ao3_wrangling_faq_link: link_to(t(".general_resources.ao3_wrangling_faq"), "https://wiki.transformativeworks.org/mediawiki/AO3_Wrangling_Project_FAQ")) %>
```
The "project" part of this link is important, so I'd prefer something like `wrangling_project_faq*` or `ao3_wrangling_project_faq*`

Bilka2 commented on app/views/tag_wranglings/index.html.erb:
```diff
@@ -1,23 +1,84 @@
+              ao3_wrangling_faq_link: link_to(t(".general_resources.ao3_wrangling_faq_link_label"), "https://wiki.transformativeworks.org/mediawiki/AO3_Wrangling_Project_FAQ")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.fandom_and_megafandom_dir_html",
```
I'd lean towards `fandom_and_megafandom_directory_html`

Bilka2 reviewed (changes requested):
The text looks good, just a few notes about locale key names now

Bilka2 reviewed (approved):
Thank you!
